pkg/controller/provider/container/ovirt/client.go

```diff
@@ -226,3 +234,365 @@ func GetInsecureSkipVerifyFlag(secret *core.Secret) bool {
 
 	return insecureSkipVerify
 }
+
+func DownloadOVFTestConection(secret *core.Secret) (ok bool, err error) {
+
+	var diskIdOVF string
+
+	url := fmt.Sprint(string(secret.Data["url"]), "/ovirt-engine/api")
```

Comment:
I didn't find where we do that (appending 'ovirt-engine/api') elsewhere, does the UI send a different URL within the secret and within the provider?

Reply:
I didn't find the saving place but it's sent with '/ovirt-engine/api', removing